src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -137,6 +134,17 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup>
+    <VsixUtilPath>$(PkgMicrosoft_VSSDK_BuildTools)\tools\vssdk\bin\VsixUtil.exe</VsixUtilPath>
+    <CorFlagsPath>$(CORFLAGS_PATH)</CorFlagsPath>
```

Comment:
I ended up modifying the build pipeline and adding "locate" step like we do for the signing tool. Msbuild was not finding it otherwise.